site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -189,10 +189,7 @@
             </div>
         {% endif %}
         {% if show_cancel_edit_form is defined and show_cancel_edit_form %}
-        <a onclick="$('#edit-user-post').css('display', 'none');
-                 $(this).closest('.thread-post-form').find('[name=thread_post_content]').val('');
-                 $('#title').val('');
-                 $(this).closest('form').trigger('reinitialize.areYouSure');"
+        <a onclick="cancelEditPostForum.call()"
```

Comment:
Works as intended, so I'll approve.
Just a small code suggestion:
```suggestion
        <a onclick="cancelEditPostForum()"
```
`<function>.call()` is only necessary when you want to invoke a function with a specific calling context (`this` value). In this case, it doesn't matter, so you can just call the function directly.